sets/sets.go

```diff
@@ -105,3 +105,15 @@ func (s *Set[T]) Intersection(x *Set[T]) *Set[T] {
 	}
 	return result
 }
+
+// Union returns a set containing all the elements of s and x.
+func (s *Set[T]) Union(x *Set[T]) *Set[T] {
```

Comment:
Don't you also need the set difference as you wrote in chat? or we already have that...

Reply:
Ah we have it... 
`func (s *Set[T]) Difference(x *Set[T]) *Set[T]`